### PR TITLE
scrub sql usernames

### PR DESCRIPTION
**Pull request by wpride**
Addresses http://manage.dimagi.com/default.asp?231582

@benrudolph

### CODE REVIEW

**Comment by benrudolph** [comment]
legit. looks good. a test for a username with `.`s might be useful

**Comment by wpride** [comment]
@benrudolph added

File: src/test/resources/requests/filter/filter_cases_sql_escape.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "filter_expr": "[@case_type = 'coverage_basic'][@status = 'open']",
+  "session_data": {
+    "domain": "test",
+    "username": "test3"
```
Comment (benrudolph) [inline src/test/resources/requests/filter/filter_cases_sql_escape.json]: Doesn't this have to have a . in it?

**Comment by benrudolph** [comment]
👍

**Pull request merged**